ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -741,11 +760,52 @@ const ControlledTierEraModule = (props: {
   ) : null;
 };
 
-const DARHeader = () => (
-  <FlexColumn style={styles.headerFlexColumn}>
-    <Header style={styles.headerRW}>Researcher Workbench</Header>
-    <Header style={styles.headerDAR}>Data Access Requirements</Header>
-  </FlexColumn>
+// the header outside the Fadebox
+const OuterHeader = (props: { pageMode: DARPageMode }) =>
+  props.pageMode === DARPageMode.INITIAL_REGISTRATION && (
+    <FlexColumn style={styles.registrationOuterHeader}>
+      <Header style={styles.regHeaderRW}>Researcher Workbench</Header>
+      <Header style={styles.regHeaderDAR}>Data Access Requirements</Header>
+    </FlexColumn>
+  );
+
+// the header inside the Fadebox
+const InnerHeader = (props: { pageMode: DARPageMode }) =>
+  props.pageMode === DARPageMode.INITIAL_REGISTRATION ? (
+    <div
+      data-test-id='initial-registration-header'
+      style={styles.pleaseComplete}
+    >
+      Please complete the necessary steps to gain access to the <AoU />{' '}
+      datasets.
+    </div>
+  ) : (
+    <FlexColumn>
+      <div
+        data-test-id='annual-renewal-header'
+        style={styles.renewalHeaderYearly}
+      >
+        Yearly Researcher Workbench access renewal
+      </div>
+      <div style={styles.renewalHeaderRequirements}>
+        <RenewalRequirementsText /> For any questions, please contact{' '}
+        <SupportMailto />.
+      </div>
+      <div style={styles.pleaseComplete}>
+        Please complete the following steps.
+      </div>
+    </FlexColumn>
+  );
+
+const SelfBypass = (props: { onClick: () => void }) => (
```

Comment:
opportunistic refactoring: strictly an extraction

Reply:
neither the config setting nor the text include "all" so my pref. would be to keep this